packages/examples/src/api/fhir/operations/bot-deploy.ts

```diff
@@ -0,0 +1,24 @@
+import { MedplumClient } from '@medplum/core';
+
+const medplum = new MedplumClient();
+
+// start-block deployTs
+await medplum.post(medplum.fhirUrl('Bot', '[id]', '$deploy').toString(), {
+  filename: 'example-bot',
```

Comment:
Make this example more real. Use the hello-patient.js (the `dist/` code) in the `code` parameter